src/Microsoft.AspNet.Mvc.Core/Filters/ProducesAttribute.cs

```diff
@@ -17,6 +17,21 @@ namespace Microsoft.AspNet.Mvc
     [AttributeUsage(AttributeTargets.Class | AttributeTargets.Method, AllowMultiple = false, Inherited = true)]
     public class ProducesAttribute : ResultFilterAttribute, IApiResponseMetadataProvider
     {
+        /// <summary>
+        /// Initializes an instance of <see cref="ProducesAttribute"/>.
+        /// </summary>
+        /// <param name="type">The type of object that is going to be written in the response.</param>
+        public ProducesAttribute([NotNull] Type type)
```

Comment:
Should there also be an overload which lets you add a type and content types as well?